scrape_meetup.py

```
@@ -0,0 +1,479 @@
+                    continue
+                
+                # Upload to Firestore
+                db.collection('scraped_events').add(event)
+                print(f"  ✓ Uploaded: {event.get('name', 'Unknown')}")
+                uploaded += 1
+                
+            except Exception as e:
+                print(f"  ✗ Error: {e}")
+        
+        print(f"\n{'='*70}")
+        print(f"Upload Summary:")
```

Ruff (0.14.4)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)